package.json

```diff
@@ -5,7 +5,7 @@
     "scripts": {
         "start": "nr1 nerdpack:serve",
         "samanez": "nr1 nerdpack:serve --profile=samanez",
-        "test": "jest -w 1",
+        "test": "jest -w 1 -u",
```

Comment:
@rsamanez This change is not mentioned in the pull request notes.

Reply:
This is an Update to resolve an issue with the Unit tests

Reply:
Using this Sample Logo from this URL:
https://www.freepnglogos.com/uploads/google-logo-png/google-logo-png-suite-everything-you-need-know-about-google-newest-0.png

Results:
![image](https://user-images.githubusercontent.com/1933521/146394554-0dd51999-bdf6-495e-a578-12f53dd2e33b.png)

Using this other Logo:
https://www.freepnglogos.com/uploads/google-logo-png/file-google-logo-svg-wikimedia-commons-23.png

Results:
![image](https://user-images.githubusercontent.com/1933521/146395054-82ee906a-ba09-4dc2-808c-8607af2259be.png)